lib/entity.js

```diff
@@ -65,6 +66,16 @@ var RESERVED_PROPERTY_NAMES = [
   'remove'
 ];
 
+/** Timeout for azure table requests */
+var AZURE_TABLE_TIMEOUT     = 30 * 1000;
+
+/** Azure table agent used for all instances of the table client */
+var globalAzureTableAgent = new https.Agent({
+  keepAlive:      true,
+  maxSockets:     5000,
+  maxFreeSockets: 1000
```

Comment:
Okay, trying with `1000` and `500`... Honestly I have no idea... but these numbers are way lower than `Infinity` which is the default... 

Note, the node 0.10 limit of 5 connections, might actually have made azure table slower for us... As could nagle which we haven't disabled with this...